_plugins/related_posts.rb

```diff
@@ -0,0 +1,62 @@
+# https://github.com/jumanji27/related_posts-jekyll_plugin
+
+require 'jekyll/post'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.